static-assets/components/cstudio-common/common-api.js

```diff
@@ -2472,16 +2477,20 @@ var nodeOpen = false,
                                                                 refreshFn(parentItemTo.item, null);
                                                             },
                                                         };
-                             
-                                                        CStudioAuthoring.Operations.editContent(
-                                                            contentTO.contentType,
-                                                            CStudioAuthoringContext.site,
-                                                            pasteResponse.status[0], 
-                                                            "", 
-                                                            pasteResponse.status[0], 
-                                                            false,
-                                                            editCb,
-                                                            new Array());
+
+                                                        if (CStudioAuthoring.Constants.IMAGE_VALID_EXTENSIONS.indexOf(extension) != -1) {
```

Comment:
What if it's a static asset of a different type?
Are we really refreshing per asset type? That's not scalable. We will get more and more types in the future.